File: app/src/test/java/com/xlythe/sample/camera/MainActivityTest.java
package com.xlythe.sample.camera;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.Robolectric;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.annotation.Config;

import static org.junit.Assert.assertNotNull;

@RunWith(RobolectricTestRunner.class)
@Config(minSdk = 33)
public class MainActivityTest {

    @Test
    public void testActivityCreation() {
        MainActivity activity = Robolectric.buildActivity(MainActivity.class).create().get();
        assertNotNull(activity);
    }
}

File: app-compose/src/test/java/com/xlythe/sample/camera/MainActivityTest.kt
package com.xlythe.sample.camera

import org.junit.Assert.assertNotNull
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.Robolectric
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(minSdk = 24)
class MainActivityTest {

    @Test
    fun testActivityCreation() {
        val activity = Robolectric.buildActivity(MainActivity::class.java).create().start().resume().get()
        assertNotNull(activity)
    }
}
